Sources/SwiftKuery/QueryError.swift

```diff
@@ -30,6 +30,8 @@ public enum QueryError: Error {
     case unsupported(String)
     /// An error in transaction.
     case transactionError(String)
+    /// Invalid table definition
+    case invalidTableDefinition(String)
```

Comment:
NOTE: This is a breaking change to the API

Reply:
Realised this is not necessary